scripts/create-minikube.sh

```diff
@@ -8,8 +8,8 @@ set -u
 # print each command before executing it
 set -x
 
-export MINIKUBE_VERSION=v1.2.0
-export KUBERNETES_VERSION=v1.14.5
+export MINIKUBE_VERSION=v1.9.2
+export KUBERNETES_VERSION=v1.18.0
```

Comment:
There was 1.18.1 is that already available to be used?

Reply:
updated